operator/pom.xml

```diff
@@ -289,6 +289,13 @@
                         </goals>
                     </execution>
                 </executions>
+                <configuration>
+                    <systemPropertyVariables>
+                        <OPERATOR_TEST_LABEL_EXPRESSION>my-value</OPERATOR_TEST_LABEL_EXPRESSION>
```

Comment:
It's out of the scope of this PR, but IMHO, the operator module contains a lot of different contexts related to the environment. I mean, we have at the same place dev, prod, test related stuff, and also the examples. IMO, it's starting to be a bit messy and could be more challenging to orient yourself to what's essential and what's not.

Maybe we could consider some follow-up tasks related to the separation of these contexts a little bit. @vmuzikar WDYT?

Reply:
@mabartos Done.